src/igvfd/schemas/sequence_file.json

```diff
@@ -296,6 +296,28 @@
                 "appscript": "ACTGGTCC",
                 "igvf_utils": "ACTGGTCC"
             }
+        },
+        "base_modifications": {
```

Comment:
Should this property be restricted to nanopore reads?

Reply:
That would probably make sense, I'll add that